.github/workflows/ci.yml

```
@@ -45,7 +45,7 @@ jobs:
         with:
           name: code-coverage-ubuntu-latest-20.x
           path: coverage/
-      - uses: SonarSource/sonarcloud-github-action@v2.2
+      - uses: SonarSource/sonarcloud-github-action@v2.3
```

```suggestion
      - uses: SonarSource/sonarcloud-github-action@v2
```

Better to use major version tags